src/YesSql.Core/Provider/IDbProviderOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using YesSql.Core.Services;
+
+namespace YesSql.Core.Provider
+{
+    public interface IDbProviderOptions
+    {
+        string ProviderName { get; set; }
```

Comment:
Look @sebastienros while we will remove the `ProviderName` property, so the `IDbProviderOptions` will be useless, let us remove it for now, and if we there's a need in the future to have provider name, table prefix .. ect, we will bring it back.

Are you agree with that?